### DISCUSSION TITLE
[UI] The button for manually folding the menu has disappeared in version 9.4.0.

### DISCUSSION THREAD

**Discussion by BertOnline** [apache/skywalking#10528] [Q&A]
The button for manually folding the menu has disappeared in version 9.4.0. Is it deliberately designed by the new version?
![image](https://user-images.githubusercontent.com/17882970/224545137-54075459-d85a-4cc3-a06d-b6006542b6ad.png)
![image](https://user-images.githubusercontent.com/17882970/224545208-f58db951-1bb4-48c2-84ea-df0f6d777cd7.png)

**Comment by wu-sheng** [comment]
Click the menu, it will pop out automatically.
